todo_app/templates/index.html

```diff
@@ -8,27 +8,44 @@ <h1 class="display-4">To-Do App</h1>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <form action="/add_item" method="post">
+    <div class="col-md-12"> 
+      <h2>New Item</h2>
+      <form action="{{ url_for('add_todo_item') }}" method="post">
+        <label for="title">Title</label>
         <input class="form-control form-control-lg" type="text" name="title" id="title" placeholder="What do you want to do?" required>
-        <label for="title"><em>Press <code>Return</code> to save</em></label>
+        <label for="description">Description</label>
+        <input class="form-control form-control-lg" type="text" name="description" id="description" placeholder="Perhaps a more detailed description?">
+        <button type="submit" class="btn btn-primary">Add</button>
+        <label for="title"><em>or press <code>Return</code> to save</em></label>
       </form>
-      <ul class="list-group mb-4">
-        {% if not items_list %}
-          <b>No items to do yet</b>
-        {% endif %}
-        {% for item in items_list %}
-          <li>
-            {% if item.status == 'Completed' %}
-              <del><a href="{{ url_for('complete_item', item_id=item.id) }}">{{ item.title }}</a></del>
+
+      <h2>Items</h2>
+      {% if not items_list %}
+        <b>No items to do yet</b>
+      {% endif %}
+      {% for item in items_list %}
+        <div class="row">
+          <div class="col-sm">
+            {% if item.status.value == 'Done' %}
+              <a href="{{ url_for('uncomplete_item', item_id=item.id) }}">
```

Comment:
Is there a reason to make the todo's name a link, when you also provide an explicit one? From a UX perspective, it's not clear what clicking on the name will do, but I'd expect it to e.g. provide more info on the card
I'd be inclined to stop this being a link, and just let the user use the buttons below